content/scout/metrics-exporter.md

```diff
@@ -170,11 +173,172 @@ Prometheus UI at <http://localhost:9090/targets>.
 To view the Grafana dashboards, go to <http://localhost:3000/dashboards>,
 and sign in using the credentials defined in the Docker Compose file (username: `admin`, password: `grafana`).
 
-![Policy dashboard in Grafana](./images/scout-metrics-grafana.png "Policy dashboard in Grafana")
+![Vulnerability dashboard in Grafana](./images/scout-metrics-grafana-vulns.png "Vulnerability dashboard in Grafana")
+
+![Policy dashboard in Grafana](images/scout-metrics-grafana-policy.png "Policy dashboard in Grafana")
 
 The dashboards are pre-configured to visualize the vulnerability and policy metrics scraped by Prometheus.
 
-### Revoke an access token
+## Datadog
+
+This section describes how to scrape the metrics endpoint using Datadog.
+Datadog pulls data for monitoring by running a customizable
+[Agent](https://docs.datadoghq.com/agent/?tab=Linux) that scrapes available
+endpoints for any exposed metrics. The OpenMetrics and Prometheus checks are
+included in the agent, so you don’t need to install anything else on your
+containers or hosts.
+
+This guide assumes you have a Datadog account, a Datadog API Key. Refer to the
+[Datadog documentation](https://docs.datadoghq.com/agent) to get started.
+
+### Configure the Datadog agent
+
+To start collecting the metrics, you will need to edit the agent’s
+configuration file for the OpenMetrics check. If you're running the agent as a
+container, such file must be mounted at
+`/etc/datadog-agent/conf.d/openmetrics.d/conf.yaml`.
+
+The following example shows a Datadog configuration that:
+
+- Specifies the OpenMetrics endpoint targeting the `dockerscoutpolicy` Docker organization
+- A `namespace` that all collected metrics will be prefixed with
+- The [`metrics`](#metrics) you want the agent to scrape (`scout_*`)
+- An `auth_token` section for the Datadog Agent to authenticate to the Metrics
+  endpoint, using a Docker PAT as a Bearer token.
+
+```yaml
+instances:
+  - openmetrics_endpoint: "https://api.scout.docker.com/v1/exporter/org/dockerscoutpolicy"
+    namespace: "scout-metrics-exporter"
+    metrics:
+      - scout_*
+    auth_token:
+      reader:
+        type: file
+        path: /var/run/secrets/scout-metrics-exporter/token
+      writer:
+        type: header
+        name: Authorization
+        value: Bearer <TOKEN>
+```
+
+> **Important**
+>
+> Do **not** replace the `<TOKEN>` placeholder in the previous configuration
+> example. It must stay as it is. Only make sure the Docker PAT is correctly
+> mounted into the Datadog Agent in the specified filesystem path. Save the
+> file as `conf.yaml` and restart the agent.
+{ .important }
+
+When creating a Datadog agent configuration of your own, make sure to edit the
+`openmetrics_endpoint` property to target your organization, by replacing
+`dockerscoutpolicy` with the namespace of your Docker organization.
+
```

Comment:
I think we should also mention that they can configure the scrape interval using this check:

https://github.com/DataDog/integrations-core/blob/master/openmetrics/datadog_checks/openmetrics/data/conf.yaml.example#L613-L617